ui/app/src/Utils/Filter.elm

```diff
@@ -22,6 +22,7 @@ import Url exposing (percentEncode)
 type alias Filter =
     { text : Maybe String
     , group : Maybe String
+    , customGrouping : Bool
```

Comment:
Whenever I look at boolean flags/query strings, I wonder if it would be better to have a concrete setting, like `grouping=default` or `grouping=custom`. It helps keep things from ballooning to ever larger sizes. For example, I'm not super thrilled with how I introduced `silenced=true` and `inhibited=true`, but I wasn't sure if it would be better or worse to have `alert_state=silenced|inhibited`.

Any thoughts on this? Stay with the boolean flag, or move use a `groups` query string? @mxinden @simonpasquier @w0rm

Reply:
@stuartnelson3 I think the missing parameter should be the default. Because this is how you open it anyways. So `grouping=default` is not really needed.